### DISCUSSION TITLE
Help with understanding webhook_prefix section.

### DISCUSSION THREAD

**Discussion by BCC-BGChatCommunity** [qaisjp/go-discord-irc#97] [Q&A] [answered]
Help with understanding webhook_prefix section. Please make me some example for this webhook_prefix becouse after 5 hours of testing different things i didn't figure out. I have the webhook and everything. What kind of prefix you put there? Char like # or ~ or link or webhook id ? Sorry if i'm stupid but is important for me :) 

FATA[0000] Go-Discord-IRC failed to start.               error="can't open discord: failed to refresh guild webhooks: could not get webhooks: HTTP 404 Not Found, {\"message\": \"Unknown Guild\", \"code\": 10004}"

Thanks

**Comment by llmII** [comment, marked as answer]
When I setup go-discord-irc I didn't manually create a webhook over on discord, but gave the bot rights to manage webhooks. I believe `README.md` explains that the bot needs that, and presence intents (forgive me if the terminology I'm using doesn't word for word match what `README.md` says).

The default in `config.yml` is fine for getting go-discord-irc running and am not even sure there is an actual need to change it (I'm running it like that across several networks all relaying to the same guild). I can't say for certain but I'd probably avoid symbols in the webhook.

**Reply by BCC-BGChatCommunity** [reply]
Thank you for the answer. But i don't get it. The default webhook_prefix: "(auto-test)" didn't work at all. My question is what kind of prefix i need to set there. Some id from discord or something else. Because i get same error again and again. 
Only this prefix make me feel i make mistake with it. If is not why i get FATA[0000] Go-Discord-IRC failed to start. error="can't open discord: failed to refresh guild webhooks: could not get webhooks: HTTP 404 Not Found, {"message": "Unknown Guild", "code": 10004}" this error. 
Thanks

**Reply by llmII** [reply]
In your config, lines 2 thru 5:

```
discord_token: abc.def.ghi
irc_server: localhost:6697
guild_id: 315277951597936640
nickserv_identify: password123
```

Unknown guild... did you change `guild_id` to the ID of your discord server? Do you have administrative privileges on the server wherein you can invite a bot? Did you create the bot and a Discord application on Discord's site?

There are several steps to enabling this to work from Discord's side.

I would suggest it's not a `webhook_prefix` issue at this point but a matter of finding out, setting up, and utilizing the Discord Server's ID and setting up a bot from Discord's site.

When you do create the application and bot on Discord's side you will need to be sure to give it webhook privileges and presence intent privileges. You'll also need to use the OAuth section to get you a link you copy/paste and then invite the bot to a server via. I'm also discussing this process from memory. Any more specific and I'd need to go to Discord's site and figure out things but this is basically a less detailed overview of the process I had to go through to set it up on Discord's side.

**Reply by BCC-BGChatCommunity** [reply]
I didn`t make difference between channel id and server id. Now i find the correct value! Great work thanks a lot!
